agent/lib/kontena/workers/service_pod_manager.rb

```diff
@@ -22,11 +21,8 @@ def initialize(autostart = true)
     end
 
     def start
-      observe(Actor[:node_info_worker]) do |node|
-        @node = node
-      end
+      @node = observe(Actor[:node_info_worker], timeout: 300.0)
```

Comment:
This is assuming that `Manager` actor is not interested in later `@node` updates...